plugins/readme/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "cem-plugin-readme",
+  "version": "0.1.0",
+  "description": "Generate README.md from custom elements sources",
+  "author": "open-wc",
+  "contributors": [
+    "Benny Powers <web@bennypowers.com>"
+  ],
+  "license": "MIT",
+  "scripts": {
+    "test": "uvu test",
+    "test:watch": "uvu -w test"
```

Comment:
uvu doesnt have a watch mode right? or did they add it?

Reply:
I think i got that from a tutorial. doesn't work anyways i'll remove